pdk/1.0/modules/ROOT/pages/policies-pdk-compile-policies.adoc

```diff
@@ -0,0 +1,41 @@
+= Compiling Custom Policies
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+You can compile the custom policy project after you complete all the steps in xref:policies-pdk-create-project.adoc[].
+
+You must compile the custom policy before you can deploy it to a Flex Gateway.
```

Comment:
I wonder if these two sentences should go in "Before You Begin"? I notice there's some redundancy. 

Also, instead of mentioning prerequisites in the intro, typically we include an intro sentence. Something like:

"Compile custom policies using the `make` tool." 

Or something that briefly describes the outcome of following the tutorial.